py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2184,6 +2184,44 @@ def aten_ops_avg_pool(
     )
 
 
+def adaptive_pool_static_input_validator(pool_node: Node) -> bool:
+    output_size = args_bounds_check(pool_node.args, 1)
+    return all([x > 0 for x in output_size])
```

Comment:
Should this check input or output? since for adaptive pooling I think the output is constant?

Reply:
According to the schema:
```
- func: adaptive_avg_pool2d(Tensor self, SymInt[2] output_size) -> Tensor
```
I think this should check `output_size`, because input is a Tensor instead of size, so its shape doesn't contain negative numbers. However, even though `output_size` is constant, but it is passed by users, so they may leave one dim as `-1`, but this is not supported because `output_size` is necessary for calculating kernel/stride sizes. 
Pytorch doesn't support negative `output_size` either, e.g., `RuntimeError: adaptive_avg_pool2d: elements of output_size must be greater than or equal to 0 but received {1, -2}`